src/backend/src/prisma/schema.prisma

```diff
@@ -109,6 +109,33 @@ enum Design_Review_Status {
   DONE
 }
 
+enum Graph_Type {
+  BAR
+  LINE
+  PIE
+}
+
+enum Data_Type {
+  CAR
+  PROJECT
+  TEAM
+  CHANGE_REQUEST
+  BUDGET
+  DESIGN_REVIEW
+  USER
+}
+
+enum Values {
+  SUM
+  AVERAGE
+}
+
+enum Permission {
```

Comment:
So I probably did a bad job of explaining permissions but permissions are more like "edit graph" and then permissions can belong to users or roles. These examples are really just pointing to roles themselves